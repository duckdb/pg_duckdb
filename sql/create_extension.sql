CREATE EXTENSION quack;
CREATE TABLE tbl(a varchar);
INSERT INTO tbl VALUES ('test');
SELECT * FROM tbl;
