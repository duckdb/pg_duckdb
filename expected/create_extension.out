CREATE EXTENSION quack;
CREATE TABLE tbl(a varchar);
INSERT INTO tbl VALUES ('test');
SELECT * FROM tbl;
INFO:  Column name: a, Type: VARCHAR
  a   
------
 test
(1 row)

